controller/proxy-injector/fake/data/pod-with-debug.patch.json

```diff
@@ -148,7 +156,7 @@
         },
         {
           "name": "LINKERD2_PROXY_DESTINATION_PROFILE_SUFFIXES",
-          "value": "."
+          "value": "svc.cluster.local."
```

Comment:
This was not previously specified but now that we are using `Values` generated from `Values.yaml` which has `.svc.cluster,local` as the value, it got added.